app/controllers/devise_token_auth/confirmations_controller.rb

```diff
@@ -71,7 +71,7 @@ def render_not_found_error
     private
 
     def resource_params
-      params.permit(:email, :confirmation_token, :config_name)
+      params.permit(:email, :confirmation_token, :config_name, :redirect_url)
```

Comment:
you didn't change the `redirect_url` definition in order to this change to make sense